frontend/src/components/RecipeDetails/RecipeDetails.jsx

```
@@ -58,11 +61,11 @@
           <Text>{recipe.likes} likes</Text>
         </HStack>
       </HStack>
-      <Separator size="lg" mb={4} />
+      <Separator mb={4} />
 
       {/* Recipe Description */}
       <div dangerouslySetInnerHTML={{ __html: recipe.summary }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)